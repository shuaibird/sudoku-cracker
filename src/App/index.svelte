<script>
    import Layout from '@Components/Layout';
    import ImageExample from '@Components/ImageExample';
    import HelloWorld from '@Components/HelloWorld';
</script>

<style src="./style.scss">

</style>

<Layout>
    <div class="container">
        <div class="container__top">
            <ImageExample />
        </div>
        <div class="container__middle">
            <HelloWorld />
        </div>
    </div>
</Layout>
