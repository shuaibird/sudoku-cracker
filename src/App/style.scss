.container {
  background-color: #2c3e50;
  display: flex;
  flex-direction: column;
  flex: 1 1 100%;
  justify-content: flex-start;
  align-items: center;
  min-height: 100vh;

  &__top {
    margin: 100px;
  }

  &__middle {
    display: flex;
    flex: 1 1 100%;
    justify-content: center;
    align-items: flex-start;
  }
}
