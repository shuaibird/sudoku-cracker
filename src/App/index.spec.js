import App from './index.svelte'
import { render } from '@testing-library/svelte'

