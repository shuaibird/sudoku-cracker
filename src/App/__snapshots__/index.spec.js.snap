// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the app 1`] = `
<body>
  <div>
    <div
      class="layout"
    >
      <div
        class="container"
      >
        <div
          class="container__top"
        >
          <div
            class="image-content"
          >
            <div
              class="bg-image"
            />
          </div>
        </div>
         
        <div
          class="container__middle"
        >
          <div
            class="title"
          >
            <span
              class="title__top"
            >
              Hello
            </span>
             
            <span
              class="title__front"
            >
              World
            </span>
             
            <span
              class="title__back"
            >
              World
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
